opensearch/examples/advanced_index_actions.rs

```diff
@@ -0,0 +1,60 @@
+use opensearch::{ http::transport::TransportBuilder, OpenSearch, http::transport::SingleNodeConnectionPool, cert::CertificateValidation};
+use url::Url;
+use opensearch::indices::{ IndicesCreateParts, IndicesClearCacheParts, IndicesFlushParts, IndicesRefreshParts, IndicesCloseParts, IndicesOpenParts, IndicesForcemergeParts, IndicesAddBlockParts, IndicesCloneParts, IndicesSplitParts, IndicesPutSettingsParts,IndicesDeleteParts};
+use serde_json::json;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Let's create a client instance, and an index named `movies`:
+
+    // Create a client to make API calls to OpenSearch running on http://localhost:9200.
+    let _client = OpenSearch::default();
+    // Alternatively, you can create a client to make API calls against OpenSearch running on a specific url::Url.
+    let url = Url::parse("https://example.com")?;
+    let transport = TransportBuilder::new(SingleNodeConnectionPool::new(url)).cert_validation(CertificateValidation::None).build()?;
+    let client = OpenSearch::new(transport);
```

Comment:
Let's make this consistent with the other guides:
```suggestion
    // Create a client to make API calls to OpenSearch running on https://localhost:9200.
    let url = Url::parse("https://localhost:9200")?;
    let credentials = Credentials::Basic("admin".into(), "admin".into());
    let transport = TransportBuilder::new(SingleNodeConnectionPool::new(url))
          .cert_validation(CertificateValidation::None)
          .auth(credentials)
          .build()?;
    let client = OpenSearch::new(transport);
```